projects/igniteui-angular/src/lib/grids/filtering/grid-filtering-row.component.ts

```diff
@@ -218,12 +218,15 @@ export class IgxGridFilteringRowComponent implements AfterViewInit {
     public onPrefixKeyDown(event: KeyboardEvent) {
         if ((event.key === KEYS.ENTER || event.key === KEYS.SPACE || event.key === KEYS.SPACE_IE) &&
             this.dropDownConditions.collapsed) {
-            this._conditionsOverlaySettings.positionStrategy.settings.target = this.inputGroupPrefix.nativeElement;
-            this.dropDownConditions.toggle(this._conditionsOverlaySettings);
+                this.toggleConditionsDropDown(this.inputGroupPrefix.nativeElement);
```

Comment:
Could you please fix the formatting of this line.

Reply:
Formatting is fixed.